components/records/custom-types.md

```diff
@@ -62,8 +62,8 @@
 
 | Type       | Variable     |
 |------------|--------------|
-| `ETerm[]`      | `types`    | 
-| `ETerm[]`   | `areas`      | 
+| `Term[]`      | `types`    | 
+| `Term[]`   | `areas`      | 
```

Comment:
please align table in the whole file
it annoys me

Reply:
formatted